config/default.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
       revisions: 'revisions',
       save: 'save',
     },
+    plugins: {
```

Comment:
Plugabilly should only care if it's an array [if `config.search` is there](https://github.com/punchcard-cms/plugabilly/blob/df72dada47e754777937c81fbd4c5c804335a0cd/lib/meta.js#L16), so we should only need Content Types to ensure that it is passing an array if it wants to pass something along